src/main/java/no/nav/veilarboppfolging/controller/AdminController.java

```diff
@@ -34,6 +53,18 @@ public String republiserTilordnetVeileder() {
         return JobRunner.runAsync("republiser-tilordnet-veileder", kafkaRepubliseringService::republiserTilordnetVeileder);
     }
 
+    @GetMapping("/hentVeilarbinfo/bruker")
+    public Veilarbportefoljeinfo hentVeilarbportefoljeinfo(@RequestParam AktorId aktorId) {
+        sjekkTilgangTilAdmin();
+
+        Optional<VeilederTilordningEntity> tilordningEntity = veilederTilordningerRepository.hentTilordnetVeileder(aktorId);
+        return new Veilarbportefoljeinfo().setVeilederId(tilordningEntity.map(x -> NavIdent.of(x.getVeilederId())).orElse(null))
+                .setErUnderOppfolging(tilordningEntity.map(VeilederTilordningEntity::isOppfolging).orElse(false))
+                .setNyForVeileder(tilordningEntity.map(VeilederTilordningEntity::isNyForVeileder).orElse(false))
+                .setErManuell(manuellStatusService.hentManuellStatus(aktorId).map(ManuellStatusEntity::isManuell).orElse(false))
```

Comment:
maybe move function calls outside...just to be more readable

Reply:
Sure 👍